projects/ui-components/lf-selection-list/lf-list-option.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, ElementRef, Input, TemplateRef, ViewChild } from '@angular/core';
+import { ILfSelectable } from '@laserfiche/lf-ui-components/shared';
+
+@Component({
+  selector: 'lf-list-option-component',
+  templateUrl: './lf-list-option.component.html',
+  styleUrls: ['./lf-list-option.component.css']
+})
+export class LfListOptionComponent  {
```

Comment:
since this component cannot be accessed by the user, can we mark it internal so the types doesn't go into the types npm package? 
```
/**
 * @internal
 */
@Component({
...
})
```

Reply:
yeah there's also a stripInternal property in the tsconfig that will strip @internal on compilation but we don't use that, we use it to help our types bundler not include stuff that won't be included in the CDN